src/libsync/networkjobs.cpp

```diff
@@ -748,7 +724,7 @@ void SimpleNetworkJob::start()
 {
     Q_ASSERT(!_verb.isEmpty());
     // AbstractNetworkJob will take ownership of the buffer
-    sendRequest(_verb, jobUrl(), _request, _device);
+    sendRequest(_verb, _request, _device);
```

Comment:
See my previous comment, too. I think there is no reason for the `verb` to change after the class's construction. It should be managed by the superclass.